src/test/java/com/epam/tc/hw7/entities/User.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw7.entities;
+
+public class User {
+    public static final User Roman = new User("Roman", "Jdi1234", "ROMAN IOVLEV");
```

Comment:
didn't fix, that is implemented in exactly same way how was in lecture

Reply:
I have a conversation with your teacher (Vladimdr Nesmelov) about this. It was found out that a mistake was made at the lecture. The DTO cannot contain such important information as credentials. Please move User to other class

Reply:
moved to separate class

> You still have many code style errors in your work. Please check the "CheckStylePlagin". ![CheckStyleErrors](https://user-images.githubusercontent.com/51859570/214880210-3387bc09-9a71-417c-906a-8408873504b5.png)

Regarding checkstyle errors - I assume that such errors are displayed when you cloan my project, because I'm sure that I've fixed all of them last time. For now all errors are also fixed
![Screenshot_645](https://user-images.githubusercontent.com/111746241/214953519-470f3433-2bf1-4dbc-ba66-0a2ca1c7851d.png)